app/assets/stylesheets/admin/views/_statistics-annoucements-summary.scss

```diff
@@ -0,0 +1,35 @@
+.app-view-statistics-announcements-summary__section {
```

Comment:
I've left a comment further down,  but it looks like this is very similar to the css for the other summary page. I wonder if we move the shared css in to one file.

Reply:
as above. ✅